.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Please explain to me, what does this settings do

Reply:
Auto generated by VSCode. We can keep them